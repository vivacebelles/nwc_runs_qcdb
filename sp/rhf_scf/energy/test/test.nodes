===2018-09-27-04:15:01 PM run nodes===
compute-0-13 
