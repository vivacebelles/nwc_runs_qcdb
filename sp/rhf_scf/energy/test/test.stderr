cp: cannot stat `/home/alolinco/nwc_runs_qcdb/sp/rhf_scf/energy/test/*.movecs': No such file or directory
cp: cannot stat `/home/alolinco/nwc_runs_qcdb/sp/rhf_scf/energy/test/*.hess': No such file or directory
cp: cannot stat `/var/scratch/alolinco/test/*.hess': No such file or directory
cp: cannot stat `/var/scratch/alolinco/test/ecce*': No such file or directory
cp: cannot stat `/var/scratch/alolinco/test/*.log': No such file or directory
cp: cannot stat `/var/scratch/alolinco/test/*.out': No such file or directory
